content/applications/general/multi_company.rst

```diff
@@ -0,0 +1,115 @@
+company name to reveal a list of all allowed companies. To switch to a different company, click on
+the company name in the drop-down menu. To enable multiple companies at once, tick the checkbox next
+to each desired company name.
+
+.. figure:: multi_company/company-access.png
+   :align: center
+   :alt: An example of the list of companies a user has access to when logged into a database.
+
+   An example of a user with access to multiple companies. The current company is My Company (San
+   Francisco), while My Company (Chicago) is also active.
+
+.. note::
+   The database may refresh after each checkbox is ticked.
+
+.. _general/active-companies:
+
+Multiple active companies
+-------------------------
+
+If more than one company is active at a time, one company is highlighted in purple, and is listed on
+the menu bar. This is the considered the *current* company.
+
+When creating a new record, the *current* company is added to the record in the *Company* field,
+except under the following circumstances:
+
+- The *Company* field for a new product, or a new contact, is left blank.
+- If there is a related document already in the system, the *Company* field on the new record
+  defaults to the same company.
+
+.. example::
+   Mitchell Admin has multiple companies enabled, but the active company is `My Company (Chicago)`.
+   When he creates a new product record, the :guilabel:`Company` field is left blank by default.
+
+   When he creates a new sales team, the :guilabel:`Company` field automatically defaults to `My
+   Company (Chicago)`.
+
+Sharing data
+============
+
+In a |mcd|, certain records are able to be utilized by all of the companies (or several, based on
+permissions).
+
+Products
+--------
+
+In an |mcd|, new products are created with the :ref:`company field <general/active-companies>`
+blank, by default. If the field remains blank, the product is shared across all companies.
+
+Contacts
+--------
+
+Similar to products, contact records are shared across companies, by default. To limit access to a
+single company, click the :ref:`company field <general/active-companies>` on a contact form, and
+select a company to assign the contact to.
+
+Inter-company transactions
+==========================
+
+The :ref:`Inter-Company Transactions <general/inter-company>` feature allows for one company in the
+database to sell or purchase goods and services from another company within the same database.
+Counterpart documents for orders and invoices can be automatically generated and synchronized,
+depending on the configuration settings.
+
+.. warning::
+   To ensure inter-company transactions are handled appropriately, certain configurations, such as
+   fiscal positions and localizations, need to be accurately assigned. See :ref:`Inter-Company
+   Transactions <general/inter-company>` for additional information.
+
+Use cases
+=========
+
+Use case #1
+-----------
```

Comment:
Suggestion to use a descriptive heading, only because "Use case #" doesnt give any context to the use case itself
```suggestion
Multinational companies
-----------------------
```